DPG/Tasks/AOTTrack/qaEfficiency.cxx

```diff
@@ -1270,8 +1298,33 @@ struct QaEfficiency {
     fillEfficiency2D("ITS-TPC_vsPt_vsEta", hPtEtaItsTpc[histogramIndex], hPtEtaGenerated[histogramIndex]);
     fillEfficiency2D("ITS-TPC_vsPt_vsEta_Trk", hPtEtaTrkItsTpc[histogramIndex], hPtEtaGenerated[histogramIndex]);
     fillEfficiency2D("ITS-TPC-TOF_vsPt_vsEta", hPtEtaItsTpcTof[histogramIndex], hPtEtaGenerated[histogramIndex]);
+    if (!doPtRadius) {
+      return;
+    }
+    auto fillEfficiencyRadius = [&](const TString effname, auto num, auto den, float minRadius, float maxRadius) {
```

Comment:
Hi @upasanasharma31 I'm not sure I understand why you need minRadius and maxRadius, isn't this already an axis in the efficiency?

Reply:
Hi @njacazio sorry I'm just trying to add these (minRadius and maxRadius) for defining the  different radii intervals.